docs/source/user-guide/vision/quickstarts/plot_detection_tutorial.py

```diff
@@ -11,7 +11,7 @@
 
 If you just want to see the output of this tutorial, jump to :ref:`observing_the_result` section.
```

Comment:
```suggestion
If you just want to see the output of this tutorial, jump to :ref:`vision_segmentation_tutorial__observing_the_result` section.
```
(added comment also in relevant place)

alternatively, you can do `#observing-the-results`__ or 'observing the results <#observing-the-results>`__
without defining the label (and delete the definition)

anyway labels are project-wide so if you do choose to leave it that way, we add a file-name-related prefix for uniqueness...